src/router.js

```diff
@@ -54,6 +56,17 @@ export default new Router({
       name: 'UserConfirmEmail',
       component: UserConfirmEmail
     },
+    {
+      path: '/forgotPassword',
+      name: 'UserForgotPassword',
+      component: UserForgotPassword,
+      meta: { transition: 'fade-in-up' }
+    },
+    {
+      path: '/resetPassword/:id/:token',
```

Comment:
Co się stanie jak sobie wpisze ręcznie id jakiegoś innego użytkownika w urlu? Mając właściwy token swój.

Reply:
API zwróci kod 400 z informacją o niewłaściwym tokenie.

Reply:
Lub z informacją o tym, że uzytkownik z tym id nie istnieje ( a nie powinno for security reasons)

Reply:
Testowałeś?

Reply:
Nie :(

Reply:
Dodam do backlogu zrobienie testów automatycznych na backendzie i froncie